setup.py

```diff
@@ -1,21 +1,21 @@
 from setuptools import setup
 
-__version__ = None   # Avoids IDE errors, but actual version is read from version.py
-exec(open('rasa_nlu/version.py').read())
+__version__ = None  # Avoids IDE errors, but actual version is read from version.py
+exec (open('rasa_nlu/version.py').read())
 
 tests_requires = [
```

Comment:
All packages listed in `requirements.txt` and `dev-requirements.txt` should also be listed here, shouldn't they?